lib/publiccloud/openstack_client.pm

```diff
@@ -0,0 +1,78 @@
+# SUSE's openQA tests
+#
+# Copyright 2021 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Helper class for OpenStack connection and authentication
+#
+# Maintainer: qa-c team <qa-c@suse.de>
+
+package publiccloud::openstack_client;
+use Mojo::Base -base;
+use testapi;
+use utils qw(file_content_replace);
+
+use constant OS_CONFIG_FILE => '/root/.config/openstack/clouds.yaml';
+use constant CLOUD_INIT_FILE => '/root/mn_jeos.cloud-init';
+
+has auth_uri => sub { get_required_var('PUBLIC_CLOUD_OPENSTACK_AUTH_URI') };
```

Comment:
```suggestion
has auth_uri => sub { get_required_var('OPENSTACK_AUTH_URI') };
```

I think `PUBLIC_CLOUD` prefix brings confusion here . let's drop it

Reply:
comments applicable to all other lines in this file

Reply:
all openstack related variables are updated to `OPENSTACK_XYZ`... I will update variables.md too